### PR TITLE
Rewrite Navbar and remove now-ui.

### PR DESCRIPTION
**Pull request by ShmoogleOsukami**
This PR rewrites the navbar to be fully responsive on mobile devices.
Mobile users should now be able to navigate the site once again.

Changes:
* Rewrote _header.htm to be a fully responsive navbar.
* Removed NowUI as it was interfering with the bootstrap classes required to make the navbar work.
* Replaced NowUI Icons with OpenIconic icons.
* Tether.js has been removed as it is not used.
* Updated Gridcoin.css to fix default text colour, along with other improvements.
* Removed all NowUI Data-background-color code
* Cleaned up the look of dropdown menus
* Fixed an issue where mobile users could not scroll while the collapsed navbar is open.
* Fixed Guide nav buttons using NowUI classes

This PR looks big but it's largely file deletions.

Visually tested on:
Desktop - Chrome, Firefox, Edge, Opera, Brave, IE11
Mobile - Chrome, Edge, Firefox, Samsung Internet Browser

I've got a mirror of the site with these changes live at [Shmoogleosukami.co.uk](https://shmoogleosukami.co.uk)

### CODE REVIEW

**Comment by RoboticMind** [comment]
1,045 lines added and 13,908 lines removed 😲

**Comment by ShmoogleOsukami** [comment]
> 1,045 lines added and 13,908 lines removed 😲

It's 90% file deletions and the open-iconic css file. Important files for review are _header.htm, _layout.htm Gridcoin.css and _head.htm i think

**Comment by RoboticMind** [comment]
Yeah that makes sense, just normally don't see line change numbers that big

**Comment by ShmoogleOsukami** [comment]
> Yeah that makes sense, just normally don't see line change numbers that big

Fixed it, I'll do file cleanup on another PR later :)

**Comment by RoboticMind** [comment]
I'd probably keep the commits if they're just cleanup

**Comment by cyrossignol** [comment]
Is the font change intentional? We may want to pull in the `font-face` rule and the `font-family` directives from the old now-ui-kit.css to keep the original Montserrat typeface.

**Comment by ShmoogleOsukami** [comment]
> Is the font change intentional? We may want to pull in the `font-family` directives from the old now-ui-kit.css to keep the original Montserrat typeface.

It's not intentional rather a side effect of now-ui removal. I'll look into it tomorrow morning.

File: _includes/_header.htm [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
-                <div class="dropdown button-dropdown invert">
-                    <a href="#pablo" class="dropdown-toggle" id="navbarDropdown" data-toggle="dropdown">
-                        <button class="btn btn-round">Community <i class="now-ui-icons arrows-1_minimal-down"></i></button>
-                    </a>
-                    <div class="dropdown-menu" aria-labelledby="navbarDropdown">
-                        <a class="dropdown-header">Gridcoin Community</a>
+            <a name="Nav"></a> <!-- Nav link, returning user to top of website -->
+            <a class="navbar-brand" href="/index.htm" rel="tooltip" title="Gridcoin - Securely rewarding BOINC computation on top of POS!" data-placement="bottom"><img src="/assets/img/grc-title2.png" alt="Gridcoin title" /></a>
+            <button class="navbar-toggler grcbtn" type="button" data-toggle="collapse" data-target="#navbarNavDropdown" aria-controls="navbarNavDropdown" aria-expanded="false" aria-label="Toggle navigation">
+                <span class="oi oi-menu"></span>
```
Comment (cyrossignol) [inline _includes/_header.htm]: Minor suggestion: we can use Bootstrap's built-in `navbar-toggler-icon` on this `<span>` instead. The SVG renders more crisply than `oi-menu`. Add the `navbar-dark` class to the `navbar-toggler` button for the image to appear.

File: _includes/_header.htm [hunk 2]
```diff
@@ -1,18 +1,20 @@
...
+            </button>
+            <div class="collapse navbar-collapse" id="navbarNavDropdown">
+                <div class="navbar-nav">
+                    <!--    Community Links    -->
+                    <div class="nav-item dropdown button-dropdown">
+                        <a href="#" class="nav-link btn rounded-pill grcbtn font-weight-bold m-1" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
```
[the remark this answers is not in the document]
Comment (ShmoogleOsukami) [reply _includes/_header.htm]: CSS is not my strong suit. Hence why I aggressively use bootstrap classes. I can move some classes around so they are not all applied to one element. Makes it a little less crazy.

File: _includes/_header.htm [hunk 3]
```diff
@@ -1,18 +1,20 @@
-<nav class="navbar navbar-toggleable-sm fixed-top navbar-transparent" color-on-scroll="75">
-    <div class="container" role="navigation">
+<nav class="navbar navbar-expand-lg fixed-top">
+    <div class="container justify-content-center">
```
[the remark this answers is not in the document]
Comment (ShmoogleOsukami) [reply _includes/_header.htm]: I managed to fix the weird shift.

File: _includes/_header.htm [hunk 4]
```diff
@@ -1,18 +1,20 @@
...
+            <a class="navbar-brand" href="/index.htm" rel="tooltip" title="Gridcoin - Securely rewarding BOINC computation on top of POS!" data-placement="bottom"><img src="/assets/img/grc-title2.png" alt="Gridcoin title" /></a>
+            <button class="navbar-toggler grcbtn" type="button" data-toggle="collapse" data-target="#navbarNavDropdown" aria-controls="navbarNavDropdown" aria-expanded="false" aria-label="Toggle navigation">
+                <span class="oi oi-menu"></span>
+            </button>
+            <div class="collapse navbar-collapse" id="navbarNavDropdown">
+                <div class="navbar-nav">
+                    <!--    Community Links    -->
+                    <div class="nav-item dropdown button-dropdown">
+                        <a href="#" class="nav-link btn rounded-pill grcbtn font-weight-bold m-1" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                            Community <span class="oi oi-chevron-bottom"></span>
```
[the remark this answers is not in the document]
Comment (ShmoogleOsukami) [reply _includes/_header.htm]: Oped for the dropdown-toggle, looks nicer.

File: _includes/_header.htm (hunk 2 above)
[the remark this answers is not in the document]
Comment (ShmoogleOsukami) [reply _includes/_header.htm]: I've managed to cut some classes down.

File: _includes/_header.htm [hunk 5]
```diff
@@ -1,18 +1,20 @@
...
-                        <a class="dropdown-header">Gridcoin Community</a>
+            <a name="Nav"></a> <!-- Nav link, returning user to top of website -->
+            <a class="navbar-brand" href="/index.htm" rel="tooltip" title="Gridcoin - Securely rewarding BOINC computation on top of POS!" data-placement="bottom"><img src="/assets/img/grc-title2.png" alt="Gridcoin title" /></a>
+            <button class="navbar-toggler grcbtn" type="button" data-toggle="collapse" data-target="#navbarNavDropdown" aria-controls="navbarNavDropdown" aria-expanded="false" aria-label="Toggle navigation">
+                <span class="oi oi-menu"></span>
+            </button>
+            <div class="collapse navbar-collapse" id="navbarNavDropdown">
+                <div class="navbar-nav">
+                    <!--    Community Links    -->
+                    <div class="nav-item dropdown button-dropdown">
```
[the remark this answers is not in the document]
Comment (ShmoogleOsukami) [reply _includes/_header.htm]: Fixed.

File: _includes/_header.htm [hunk 6]
```diff
@@ -1,18 +1,20 @@
-<nav class="navbar navbar-toggleable-sm fixed-top navbar-transparent" color-on-scroll="75">
```
[the remark this answers is not in the document]
Comment (ShmoogleOsukami) [reply _includes/_header.htm]: I personally prefer the fixed background color look. What do @barton2526 @RoboticMind think. Should we maintain the fade-on-scroll or go with a fixed navbar colour?

**Comment by ShmoogleOsukami** [comment]
Since this is looking very close to being done, Pending any little tweeks or fixes, I'm going to put it up for full review now.
One problem I have observed and I'm unsure how to fix is the Navbar doesn't work correctly on Samsung Native Browser, Specific I've tested on my Galaxy S7 Edge. It's behaving weirdly but it's only on this browser. 
Chrome, Edge, Firefox all work fine.

File: _includes/_header.htm [hunk 7]
```diff
@@ -1,18 +1,22 @@
...
-                    <a href="#pablo" class="dropdown-toggle" id="navbarDropdown" data-toggle="dropdown">
-                        <button class="btn btn-round">Community <i class="now-ui-icons arrows-1_minimal-down"></i></button>
-                    </a>
-                    <div class="dropdown-menu" aria-labelledby="navbarDropdown">
-                        <a class="dropdown-header">Gridcoin Community</a>
+            <div class="collapse navbar-collapse" id="navbarNavDropdown">
+                <div class="navbar-nav">
+                    <!--    Community Links    -->
+                    <div class="nav-item dropdown">
+                        <a href="#" class="nav-link dropdown-toggle btn grcbtn" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
```
[the remark this answers is not in the document]
Comment (ShmoogleOsukami) [reply _includes/_header.htm]: Done.

File: assets/css/gridcoin.css [hunk 8]
```diff
@@ -4,22 +4,45 @@ body {
     margin-top: 0px;
     padding-top: 0px;
     background-color: #1A2632;
+    color: white;
+    font-family: "Montserrat";
```
[the remark this answers is not in the document]
Comment (ShmoogleOsukami) [reply assets/css/gridcoin.css]: Fixed. :)

File: _includes/_header.htm [hunk 9]
```diff
@@ -23,127 +27,120 @@
...
+                            <!--
+                                <a class="dropdown-item" href="https://bitshares.openledger.info/#/market/OPEN.GRC_OPEN.BTC"><i class="fa fa-exchange"></i> OpenLedger</a>
+                            -->
+                            <a class="dropdown-item" href="https://www.bitker.com/#/bbTrades/grc_btc">Bitker</a>
+                            <a class="dropdown-item" href="https://bittrex.com/Market/Index?MarketName=BTC-GRC">Bittrex</a>
+                            <a class="dropdown-item" href="https://flyp.me/">Flyp.Me</a>
+                            <a class="dropdown-item" href="https://market.rudex.org/#/asset/RUDEX.GRC">RUDEX</a>
+                            <a class="dropdown-item" href="https://www.southxchange.com/Market/Book/GRC/BTC">SouthXchange</a>
+                            <div class="dropdown-divider"></div>
+                            <h6 class="dropdown-header text-center">3rd party wallets</h6>
```
[the remark this answers is not in the document]
Comment (ShmoogleOsukami) [reply _includes/_header.htm]: Fixed, I actually didn't notice this had changed from the original site. xD Makes it a ton more readable.

**Pull request merged**